### PR TITLE
Adrv9009 iiod application

### PR DESCRIPTION
**Pull request by cristiantpop**
adrv9009: Add iio application.

This will allow reading/writing and parametrization of axi_adc and axi_dac
modules through UART, using IIO-Oscilloscope application.

### CODE REVIEW

**Comment by mchindri** [comment]
you uploaded by mistake the binary projects/adrv9009/adrv9009_zcu_706.hdf

**Comment by cristiantpop** [comment]
V2:

- remove hdf file.

File: iio/iio_app/iio_axi_adc_app.h [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+/******************************************************************************/
+
+/**
+ * struct iio_axi_adc_desc - Application desciptor.
+ * @iio_axi_adc_inst - iio ADC device handle.
+ * @iio_axi_dac_inst - iio DAC device handle.
+ * @iiod - iiod handle.
+ * @irq_desc - IRQ device.
+ * @uart_device - UART device.
+ */
```
Comment (dbogdan) [inline iio/iio_app/iio_axi_adc_app.h]: This struct has only one member - please update the comment.

File: iio/iio_app/iio_axi_dac_app.c [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+/******************************************************************************/
+/************************ Functions Definitions *******************************/
+/******************************************************************************/
+
+/**
+ * iio_axi_dac_init() - Application for reading/writing and parameterization of
+ * axi_dac device.
+ * @desc - Application descriptor.
+ * @init - Application configuration structure.
```
Comment (dbogdan) [inline iio/iio_app/iio_axi_dac_app.c]: Here we should have "@param".

File: iio/iio_app/iio_axi_dac_app.h [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+/******************************************************************************/
+
+/**
+ * struct iio_basic_app_desc - Application desciptor.
+ * @iio_axi_adc_inst - iio ADC device handle.
+ * @iio_axi_dac_inst - iio DAC device handle.
+ * @iiod - iiod handle.
+ * @irq_desc - IRQ device.
+ * @uart_device - UART device.
+ */
```
Comment (dbogdan) [inline iio/iio_app/iio_axi_dac_app.h]: This struct has only one member - please update the comment.

**Comment by dbogdan** [comment]
Besides some comments that must be updated, it looks good.

**Comment by cristiantpop** [comment]
V3:

- Fix comments.

**Pull request merged**